src/style/ngm-toolbox.css

```diff
@@ -30,20 +30,16 @@
 }
 
 .ngm-aoi-content .ui.buttons {
-  width: 95%;
+  flex: auto;
```

Comment:
Buttons can have a different size because of `flex: auto` - 
![image](https://user-images.githubusercontent.com/51954170/107359740-b9a75500-6add-11eb-8dfe-f63ebcacaa2d.png)